lib/drivers/medtronic600/docs/packetStructure.md

```diff
@@ -16,11 +17,12 @@ All other messages are broken down to USB packets before being sent on the HID i
 
 ### MiniMed Message
 A **MiniMed Message** is broken up into [USB Packet](#usb-packet)s.
+
 ![MiniMed Message](images/svg/MiniMedMessage.svg)
 
 | Field     | Bytes | Data Type  |   Comments   |
 |-----------|:-----:|:----------:|--------------|
-| Header    | 2     | Bytes      | `0x5103`     |
+| Header    | 2     | Bytes      | `0x5103` (`0x03` is an NGP device)     |
```

Comment:
Maybe mention what an NGP device is, as it's the first mention of it in this doc.